src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -60,6 +60,11 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     <% } %>
     <a href="/about.jsp">About</a>
     <a href="/testdata">Load Test Data</a>
+    <%
+      if (user != null) {
+    %>
+      <a href="/login">Logout</a>
+    <% } %>
```

Comment:
Why didn't you just wrap this in the similar if statement above?

Reply:
Because I thought log out should be the last button in the nav bar. Where do you think it should be?

Reply:
I agree log out should be the last button in nav bar.